src/test_details.ts

```diff
@@ -0,0 +1,7 @@
+import * as vscode from "vscode";
+
+type TestDetails = {
+  command: string[];
+};
+
+export const test_details = new WeakMap<vscode.TestItem, TestDetails>();
```

Comment:
This is one way we can handle the shared state. Or we can create in in _extension.ts_ and pass it into every function that will need it.

Reply:
This is fine.